airflow-core/src/airflow/ui/public/i18n/locales/nl/hitl.json

```diff
@@ -12,11 +15,13 @@
   "requiredActionCount_other": "Vereiste acties ({{count}})",
   "requiredActionState": "Vereiste actie status",
   "response": {
+    "created": "Reactie aangemaakt op ",
```

Comment:
```suggestion
    "created": "Reactie aangemaakt op",
```

Reply:
Hey @BasPH! This space is also present in the English locale, so I kept it in for consistency.  Not sure why it is there